File: app/src/main/java/com/seanmiller/alpacadashboard/TimelineActivity.java
package com.seanmiller.alpacadashboard;

import android.app.Activity;
import android.os.Bundle;

import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;

public class TimelineActivity extends Activity {
    private SwipeRefreshLayout mSwipeRefreshLayout;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mSwipeRefreshLayout = findViewById(R.id.refresh);
    }

}

File: app/src/main/java/com/seanmiller/alpacadashboard/TimelineActivity.kt
package com.seanmiller.alpacadashboard

import android.app.Activity
import android.os.Bundle
import androidx.swiperefreshlayout.widget.SwipeRefreshLayout

class TimelineActivity : Activity() {
    private var mSwipeRefreshLayout: SwipeRefreshLayout? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mSwipeRefreshLayout = findViewById(R.id.refresh)
    }
}